common/README.md

```diff
@@ -1,140 +0,0 @@
-# common
```

Comment:
Hah, wow this is old!

What an ancient relic you've found! Even the travis file contains `go 1.2`!